core/templates/dev/head/components/forms/RequireIsFloatDirective.js

```diff
@@ -16,9 +16,7 @@
  * @fileoverview Directives for schema-based form builders.
  */
 
-/* eslint-enable angular/directive-restrict */
-
-// This should come before 'apply-validation', if that is defined as
+// This should come before all validators, if that is defined as
```

Comment:
> > [seanlip] Should this be a note for validators in general, not just for this validator?
>
> [lilithxxx] Done

What exactly did you do in order to fix this? https://github.com/oppia/oppia/pull/6202#discussion_r261537628

Reply:
I am sorry but I am not understanding what to do. I edited the note above. What more should I do?

Reply:
I'm not sure you understand the note :)

I think there's another directive called applyValidation and that the original comment is saying that require-is-float should be declared before apply-validation. I was asking whether this needs to be the case for the other validation filters as well.

Reply:
Oh no I understood what the note meant, I just thought that you put a statement that this note should be for all validators and not asking a question. Well sorry about that. I checked and its only used before apply-validation directive so I think this note is fine